.github/workflows/tests.yml

```diff
@@ -0,0 +1,46 @@
+name: figures tests
+
+on: push
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - python: 2.7
+            tox-env: py27-ginkgo
+          - python: 2.7
+            tox-env: py27-hawthorn
+          - python: 2.7
+            tox-env: py27-hawthorn_multisite
+          - python: 3.5
+            tox-env: py35-juniper
+          - python: 3.8
+            tox-env: py38-juniper
+          - python: 3.8
+            tox-env: lint
+          - python: 3.8
+            tox-env: edx_lint_check
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python }}
+      - name: install dependencies
+        run: |
+          pip install tox flake8
+      - name: Uses Node.js
```

Comment:
I copied the steps that get run directly from TravisCI, so it runs `yarn` in the frontend directory before running the tox tests here just as it did on Travis. The python tests don't appear to depend on the frontend stuff in any way though. If we wanted to speed the tests up a little bit, it seems to me like we should pull out the JS stuff into its own separate test config and run that independently of the python tests.